README.md

```diff
@@ -41,12 +41,11 @@ We have [Jest](https://facebook.github.io/jest) unit tests that can be run with
 
 ### End-to-end tests
 
-We use [Cypress](https://www.cypress.io/) for our end-to-end tests. For running the tests locally we need two terminals running:
+We use [Cypress](https://www.cypress.io/) for our end-to-end tests.
+`npm run test:e2e` This will run the application and the Cypress tests in two terminals. After tests are run, it will stop all Razzle processes (we use Razzle for our application).
```

Comment:
Maybe we could reword the "two terminals" part of this sentence - The command will run the application as a "background process" which allows for the e2e's to be run in a single terminal.